CMakeLists.txt

```diff
@@ -127,6 +127,11 @@ if (WIN32)
     configure_file("scap-workbench.wxs.in" "${CMAKE_CURRENT_BINARY_DIR}/scap-workbench.wxs")
 endif()
 
+if (${CMAKE_C_COMPILER_ID} STREQUAL "GNU")
```

Comment:
let's check CXX COMPILER and check it for GNU or Clang, Clang has the same options AFAIK.